test/index.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const expect = require('chai').expect;
+const jsdom = require('jsdom');
+
+describe('node acceptance tests', function() {

```

Comment:
I don't believe google prescribes to this style:
https://github.com/googlevr/webvr-polyfill/blob/master/.jscsrc#L2
https://github.com/jscs-dev/node-jscs/blob/master/presets/google.json#L54-L56